gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.4-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.8-all.zip
```

Comment:
we could switch to binary artifact, I don't see any reason why all is needed

Reply:
It was all before, so I did not want to change it. Plus, from what I understand, all includes sources/javadocs, which makes IDE experience a bit better when interacting with Gradle scripts.